index.html

```diff
@@ -1,12 +1,1449 @@
+                </defs>
+            </svg>
+        </div>
+        <ul class="menu-items">
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle">Audit</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">Manage</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">Secure</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">Automate</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">Resource</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">About</a>
+            </li>
+            <li> <button class="btn">Book a Demo</button> </li>
+        </ul>
+    </header>
+    <main>
+        <div class="services d-flex align-items-center">
+            <div class="services-content">
+                <h1>Maximize Your IT Investment</h1>
+                <p>We will help you to audit, manage, automate and secure your entire business.
+                    <span>Schedule a demo with our specialists</span>
+                </p>
+                <div class="form-check-group d-flex">
+                    <div class="form-check">
+                        <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios1"
+                            value="option1" checked>
+                        <label class="form-check-label" for="exampleRadios1">Cybersecurity</label>
+                    </div>
+                    <div class="form-check">
+                        <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios2"
```

Comment:
refactor whole code

Reply:
done

Reply:
relevant classes are being used throughout the code